mtp_cashbook/apps/cashbook/forms.py

```diff
@@ -10,9 +10,12 @@
 from form_error_reporting import GARequestErrorReportingMixin
 from moj_auth.api_client import get_connection
 from mtp_utils.api import retrieve_all_pages
+from mtp_utils.forms import replace_default_error_messages
 
 from .form_fields import MtpTextInput, MtpDateInput
 
+replace_default_error_messages()
```

Comment:
If we are really replacing default messages everywhere, it might be better to put this in ```settings``` or in the ```__init__.py``` of the base package rather than in every ```forms.py```. I mean I'm not sure if any of our client apps actually have more than one ```forms.py``` but you know.